PlayerSkeleton.java

```diff
@@ -1,22 +1,217 @@
 
+import java.util.Arrays;
+
 public class PlayerSkeleton {
 
+	private int[][] pWidth;
+	private int[][] pHeight;
+	private int[][][] pBottom;
+	private int[][][] pTop;
+
 	//implement this function to have a working system
 	public int pickMove(State s, int[][] legalMoves) {
-		// System.out.println("Heuristic score = " + Heuristic.evaluate(s));
-		return 0;
+		
+		//Step 1. Get a copy of the current board configuration.
```

Comment:
As per my previous comment, can just make it a copy of the state, don't need to dive into parameters of the State class.